api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -72,30 +72,22 @@ public Profile getProfile(User user) throws ApiException {
     if (user.getAuthorities() != null) {
       profile.setAuthorities(new ArrayList<>(user.getAuthorities()));
     }
-    String userEmailVerificationStatus = null;
-    if (user.getEmailVerificationStatus().equals(EmailVerificationStatus.PENDING)) {
-      // Verification is pending, need to query mailchimp.
+    // Verification is pending or unverified, need to query mailchimp.
```

Comment:
If a user puts in an invalid contact email, is this going to repeatedly query mailchimp each time they get their profile?

Reply:
"Invalid" defined how? Mailchimp verifies but does not validate.

Reply:
If I put in `mydummyemailthatisntanemail`, will it sit in pending and query forever? (I guess that was a problem before? But where is the case after this PR where it will ever be unverified?

Reply:
I'm not sure why we'd ever have to query when status isn't PENDING....

Reply:
I changed the default value for email_verification_status from 1 to 0, so before the first successful call to MailChimp, the status will be UNVERIFIED.

Reply:
OK. We still don't need to query MailChimp until we sent an email and the status is pending?

Reply:
We don't send the email; we send the new email address to MailChimp – it sends the email and sends us back the status of 'pending'.

Reply:
Let me clarify: on line 79, you're checking the status of the user. That is only necessary when we have already asked MailChimp to send an e-mail for us, at which point we will set the status of the user to pending. 

So the check you had here before -- comparing verification status to PENDING -- seemed more appropriate than this check. (Unless you're worried about the case where MailChimp sends an email but we never write to our database?)

Reply:
Discussed offline. The only time you're in an UNVERIFIED state is if your account was created but the call to verify the email in MailChimp failed... it might have partially succeeded, so we can ask in that case.  UNSUBSCRIBED and CLEANED don't apply to our situation. So never mind. :)